lib/src/templates/template.dart

```diff
@@ -52,13 +52,21 @@ class DartPkgTemplate extends Template {
       await Flutter.pubGet(cwd: outputDir.path);
       installDependenciesDone();
     }
+    final isDartInstalled = await Dart.installed();
```

Comment:
This code is being replicated on multiple locations, maybe it would be worth to add it to a generic method so we don't repeat ourselves?

Something `applyFixIfPossible` (not sure if a good name, but maybe something to that matter)

Reply:
yup I'm in the process of refactoring the post generation step to be a bit more DRY 👍